Firestore/Example/Tests/Integration/API/FIRValidationTests.mm

```diff
@@ -392,18 +392,18 @@ - (void)testQueryWithNonPositiveLimitFails {
 
 - (void)testNonEqualityQueriesOnNullOrNaNFail {
   FSTAssertThrows([[self collectionRef] queryWhereField:@"a" isGreaterThan:nil],
-                  @"Invalid Query. Nil and NSNull only support equality comparisons.");
+                  @"Invalid Query. You can only perform equality comparisons on null values.");
```

Comment:
Related to one of the previous PRs: I find the existing error messages to be more readable.